### ISSUE TITLE
error when running notebook in gplugins

### ISSUE THREAD

**Issue by joamatab** [paulxshen/Luminescent.jl#43]
I had to comment out the execution of the notebook as it doesn't seem to work for me locally

![image](https://github.com/user-attachments/assets/0fe1f493-4eba-4824-bec9-1a0948b24e7f)

**Comment by paulxshen** [comment]
Yeah currently Julia binaries are manually added - lemme get back to you in few days

**Comment by joamatab** [comment]
You can take a look on femwell team does it


https://github.com/HelgeGehring/femwell


@HelgeGehring
